domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -3086,19 +3085,31 @@ public Future<RowSet<Row>> select(String sql) {
   }
 
   /**
-   * Run a select query using the readonly connection.
-   *
```

Comment:
`Run an SQL statement that returns data, for example {@code SELECT} or {@code UPDATE ... RETURNING ...}.`

Reply:
I resolved this already in the latest commit but Github doesn't recognize the differences.